readthedocs/builds/managers.py

```diff
@@ -225,3 +225,19 @@ def add_rule(
             action_arg=action_arg,
         )
         return rule
+
+
+class AutomationRuleMatchManager(models.Manager):
+
+    def register_match(self, rule, version, max_registers=15):
```

Comment:
I think 15 is not enough for a good debugging if you have several versions and a few rules. However, adding more than 15 may require pagination on this page and I suppose we want to avoid that for now.

Reply:
I think is good, usually you only push or delete one version, at least if you are doing a bulk deletion/creation you don't need more than 15.

Reply:
Even if you are not doing bulk. Just working on a team with a few people creating branches that match a few rules, you loose the recent activity that is interesting for the case you want to check.